test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -25,8 +25,8 @@ public void execute() throws IllegalValueException {
         //same word, same case: matched
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
-        //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        //same word, different case: matched
+        assertFindCommandBehavior(new String[]{"aMy"}, Arrays.asList(td.amy));
```

Comment:
It would be good if you could add more test cases to thoroughly test the case insensitivity. However, don't worry about it too much as test case design will be covered at a later point.

Reply:
Noted, thank you!